lib/AST/ASTMangler.cpp

```diff
@@ -4533,7 +4533,7 @@ void ASTMangler::appendAnyProtocolConformance(
       return;
 
     auto path = genericSig->getConformancePath(conformingType,
-                                               conformance.getAbstract());
```

Comment:
Let's add a FIXME here to remind ourselves that the conformingType parameter to this function is no longer necessary

Reply:
Sure.